pkg/schema/di.go

```diff
@@ -15,6 +17,33 @@ var cr *CoreRegistry
 // atomic guard on registry
 var regguard int32
 
+var schm *runtime.Scheme = runtime.NewScheme()
+
+var (
+	groupName    = "grafana.ap.group"
+	groupVersion = "v1"
+	once         sync.Once
+)
+
+func getCoreRegistry() *CoreRegistry {
+	once.Do(func() {
```

Comment:
i do think i like this more than directly referencing the `cr` package var...though i can't really see a reason why? maybe it's safer init-wise